tests/test_views_search.py

```diff
@@ -318,14 +358,14 @@ def test_from_parameter_invalid_pagination(
         assert errors == {('from', 'Must be at least 1.'), } or \
             errors == {('from', 'Must be greater than or equal to 1.'), }
 
-        res = client.get(search_url, query_string={'size': 1, 'from': 9999})
+        res = client.get(search_url, query_string={'size': 1, 'from': 10000})
```

Comment:
Not a blocker for me, but I have a slight doubt about this one in terms of behavior. If I request `size: 1, from: 1` I would expect to get 1 result back (i.e. result no. 1 out of 10.000). Keep in mind that it's not possible to do `size: 1, from: 0` since `from` has to be greater that 0 (there's a test above for that). If I wanted to get the results no. 10.000 I would in the same fashion do `size: 1, from: 10000`, which would return only one record, the last one.

Reply:
This is just an issue with the demo data not having 10K results. I have updated the tests hopefully in a more understandable way.